content/scaling.md

```diff
@@ -4,21 +4,45 @@ order: 40
 description: Learn how to improve your app’s performance by scaling
 ---
 
-Scaling and sizing your application is specific to the architecture and performance characteristics of your app. As apps grow they often require more resources to perform well. Galaxy accommodates growth by offering vertical and horizontal resource scaling. Scaling with Galaxy happens without downtime so your users will not experience service interruption.
+Scaling and sizing your application is specific to the architecture and performance characteristics of your app. As apps grow they often require more resources to perform well. Galaxy accommodates growth by offering vertical and horizontal resource scaling.
+
+As long as you maintain one continously running container at all times, scaling with Galaxy happens without downtime, so your users will not experience service interruption.
 
 <h3 id="what-to-look-for">What to look for</h3>
 
-We recommend that you scale when your app’s container usage exceeds either 70% memory, 80% CPU, or consistently spikes to 100% CPU. Use Galaxy’s performance graphs to track your resource usage over time.
+We recommend that you scale when your app’s container usage exceeds either 70% memory, 80% CPU, or consistently spikes to 100% CPU. Use Galaxy’s performance graphs to track your resource usage over time. 
 
 <img src="images/email-galaxy-performance-graphs-600x468.jpg" style="">
 
 <h3 id="how-to">How to scale</h3>
 
-By default new applications are deployed to Compact containers. Compact containers are great for small apps. For large codebases we recommend that you run your app on Standard containers for best performance.
+By default, new applications are deployed to Compact containers. Compact containers are appropriate for small apps. For large codebases, we recommend that you run your app on Standard containers.
+
+You may need to change your garbage collection settings for optimal performance. Garbage collection, in this context, refers to automatic memory management; memory occupied by objects that are no longer in use by the program can be freed up for general use, though the required checks can come at a performance cost.
```

Comment:
OK this text is pretty good overall but lemme tweak a few bits.

You may need to change Node's garbage collection settings for optimal performance. Garbage collection, in this context, refers to automatic memory management; memory occupied by objects that are no longer in use by the program will be freed up for general use, though the required checks can come at a performance cost.

Node (the JavaScript engine that powers Meteor) has a powerful garbage collector which can be tuned via various command-line options.  One of the most important knobs is the "max old space size" setting, which tells Node how much memory to allow the largest part of the memory heap to grow to.  Node's default (in our environment) is to allow slightly over 1 GB (1400 MB) of memory for this space.

You can control this option (and more generally, any Node setting that can be controlled at the command-line) by  setting the environment variable `$NODE_OPTIONS` in your settings.json (link to settings.json page???). The value of the `--max-old-space-size` flag is an integer number of megabytes.  For example, to allow 2000MB for the old space size, set `NODE_OPTIONS` to `"--max-old-space-size=2000"`.

Note that while this is generally the largest part of a Node process's memory usage, it does not control the Node process's entire memory footprint. Additionally, if your process forks other subprocesses they will use their own memory within your container's memory limits.

Users of Double and Quad containers may want to set this flag higher than its default of 1400 MB to take full advantage of their container's memory. Conversely, users of Standard and Compact containers may want to lower this setting to ensure that their Node processes do their best to collect enough garbage to stay inside the container limits.

We are currently investigating what the best default value for this flag should be for each of our container types so that we can set a better default for our users.  We'd love to hear feedback (via the support interface) from users as to what values work best for them.  (maybe there's a better way to write this)

also maybe this whole section should be its own subsection on the page and show up in the TOC etc